insights/parsers/tests/test_xfs_info.py

```diff
@@ -149,3 +149,10 @@ def test_ext_log_xfs_info(self):
         # Calculated information checks
         assert xfs_obj.data_size == 262144 * 4096
         assert xfs_obj.log_size == 25600 * 4096
+
+    def test_bad_xfs_info(self):
+        xfs_obj = xfs_info.XFSInfo(context_wrap("xfs_info: bad command or file name"))
+        assert hasattr(xfs_obj, 'xfs_info')
+        assert xfs_obj.xfs_info == {}
+        assert not hasattr(xfs_obj, 'data_size')
```

Comment:
For these XFSInfo which does not contain any `data` or `log` section, how about leave these `_size` as zero?
Otherwise, we'd better explain in the `Attributes` docstring that these attributes might do not exist.

Reply:
Yep, done.  Thanks @xiangce